python/doc_generator.py

```diff
@@ -47,13 +76,51 @@ def _get_transform_markdown(transform_yaml: Dict,
         md.h2('Parameters'),
         md.table(transform_yaml['arguments']),
         md.h2('Example'),
+        md.text(DATASET_PREVIEW_TEXT) if 'preview-data' in transform_yaml else None,
+        md.python_code(DATASET_PREVIEW_CODE) if 'preview-data' in transform_yaml else None,
+        t1,# md.text(transform_yaml['preview-data']) if 'preview-data' in transform_yaml else None,
+        md.text(TRANSFORMED_DATA_PREVIEW_TEXT) if 'preview-data' in transform_yaml else None,
         md.python_code(transform_yaml['example_code']),
```

Comment:
One thing about the example code...

I like it would be ideal to have a `1 to 1` relationship of the example code to create the example output df and the one shown.

Since we are generating the code and executing the transform to get the df, can't we just generate a nicely formatted string of python code as well?

This would also make writing the YAML easier, as someone won't need to supply `transform_yaml['example_code']` anymore.